### PR TITLE
Product review

### PR DESCRIPTION
**Pull request by DmitriyUshkvarok**
no server-side logic (endpoints are not ready) no styles for mobile devices (styles are not ready in the layout )written logic for the list of comments (local hard-code) written logic for uploading more comments (hard-code locally)written custom rating(without server)written custom rating and its state(without server)written form of comment sending and rating selection(without server)

### CODE REVIEW

File: src/components/Review/CommentsList/CommentsList.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      </ul>
+
+      {showLoadMore && (
+        <Button onClick={loadMoreComments} className='flex items-center justify-center rounded-[47px] w-[278px] ml-auto mr-auto bg-secondary font-medium text-[18px] text-primary'>Show more</Button>
+      )}
+    </>
+  )
+}
+
+export default CommentList
```
Comment (iakivpekarskyi) [inline src/components/Review/CommentsList/CommentsList.tsx:97]: nit: we usually go with the "Default Export (Export as function)" style for components in our project. Noticed here and in all the other components "Named Export (Export as const)" was used—any particular reason for the switch? Just want to make sure we're keeping it consistent across the board. 😊

File: src/components/Review/ReviewRatingFilter/ReviewRatingFilter.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+          ))
+
+          return (
+
+            <label key={value} className='flex items-center gap-2 cursor-pointer relative'>
+              <input
+                className='w-6 h-6 appearance-none bg-secondary rounded-[4px] cursor-pointer checked:bg-inverted'
+                type="checkbox"
+                checked={rating === value}
+                onChange={() => handleCheckboxChange(value)}
```
Comment (iakivpekarskyi) [inline src/components/Review/ReviewRatingFilter/ReviewRatingFilter.tsx:42]: nit: consider adding aria-label attributes  for screen readers.

Comment (DmitriyUshkvarok) [reply src/components/Review/ReviewRatingFilter/ReviewRatingFilter.tsx:42]: ок

File: src/components/Review/CommentsList/CommentsList.tsx (hunk 1 above)
Comment (DmitriyUshkvarok) [reply src/components/Review/CommentsList/CommentsList.tsx:97]: write it down in your notebook of ideas and someday we'll harmonize the project.

**Review by Sunagatov — APPROVED** [review]

**Pull request merged**